metadata/UiPathDocumentOCR_CPU__1__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "changeLog": "Model: 22.2.4",
+  "cpu": 0,
+  "description": "Machine Learning model(available in Preview) for extracting text, checkboxes/radio buttons and English handwriting from Documents. This model is optimized and will only run on CPU. Please see more details in the documentation page here: https://docs.uipath.com/document-understanding/docs/ml-packages.",
+  "displayName": "UiPathDocumentOCR_CPU",
+  "gpu": 0,
+  "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the UiPath Document OCR activity from OCR.Activities pack in the Official feed. File formats accepted include pdf, tiff, jpg or png files.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON37_DU",
+  "name": "UiPathDocumentOCR_CPU",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "CPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
+  "languageVersion": 3,
+  "version": 1,
```

Comment:
is this version correct?

Reply:
Yes, it is the first version optimized for CPU